static/templates/settings_tab.handlebars

```diff
@@ -94,6 +94,15 @@
       <div class="display-settings-form">
 
         <div class="control-group">
+          <label for="default_language" class="control-label inline">{{t "Default Language" }}:</label>
```

Comment:
I tried to change languages and noticed that label `Default Language` is always in English.
![screenshot from 2016-07-02 11 39 17](https://cloud.githubusercontent.com/assets/6594924/16538739/d1e2ca54-404a-11e6-8edf-e560e89b78d1.png)